programs/helium-entity-manager/src/instructions/temp_update_iot_operations_fund_metadata.rs

```diff
@@ -0,0 +1,88 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use helium_sub_daos::DaoV0;
+use mpl_token_metadata::state::{Creator, DataV2};
+use shared_utils::token_metadata::{update_metadata_accounts_v2, UpdateMetadataAccountsV2};
+
+pub const IOT_OPERATIONS_FUND: &str = "iot_operations_fund";
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct TempUpdateIotOperationsFundMetadataArgs {
+  pub metadata_url: String,
+}
+
+#[derive(Accounts)]
+pub struct TempUpdateIotOperationsFundMetadata<'info> {
+  #[account(mut)]
+  pub payer: Signer<'info>,
+  pub authority: Signer<'info>,
+  #[account(
+    has_one = authority
+  )]
+  pub dao: Box<Account<'info, DaoV0>>,
+  /// CHECK: Signs as a verified creator to make searching easier
+  #[account(
+    seeds = [b"entity_creator", dao.key().as_ref()],
+    bump,
+  )]
+  pub entity_creator: UncheckedAccount<'info>,
+  pub mint: Box<Account<'info, Mint>>,
+
+  #[account(
+    mut,
+    seeds = ["metadata".as_bytes(), token_metadata_program.key().as_ref(), mint.key().as_ref()],
+    seeds::program = token_metadata_program.key(),
+    bump,
+  )]
+  /// CHECK: Checked by cpi
+  pub metadata: UncheckedAccount<'info>,
+
+  /// CHECK: Verified by constraint
+  #[account(address = mpl_token_metadata::ID)]
+  pub token_metadata_program: AccountInfo<'info>,
+}
+
+pub fn handler(
+  ctx: Context<TempUpdateIotOperationsFundMetadata>,
+  args: TempUpdateIotOperationsFundMetadataArgs,
+) -> Result<()> {
+  let entity_creator_seeds: &[&[u8]] = &[
```

Comment:
Add a macro

Reply:
Good call out, would do if this wasn't a temp endpoint